packages/app/src/elements/unlock.ts

```diff
@@ -215,6 +215,11 @@ export class Unlock extends StartForm {
             if (e.code !== ErrorCode.DECRYPTION_FAILED) {
                 throw e;
             }
+            if (app.account?.settings.failedLoginAttemptNotifications && this._failedCount > 3) {
```

Comment:
The number is `3`, but since it starts with 0 and is only incremented _after_  the check, this _actually_ happens at the 5th failed attempt. I considered changing the whole code in this function for the numbers to add up, but deemed it out of scope.

Reply:
I think this is a completely distinct feature from failed login notifications and should probably be part of a different PR. Aside from that, we should introduce a separate setting for this, since it really doesn't have anything to do with failed **login** attempts (we're talking about **unlocking** here, and the behaviour goes beyond just a notification). I'd say let's defer this feature to a different PR, which also means the change to the `revokeSession` endpoint is unnecessary.